forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/task/FormConnectorListener.java

```diff
@@ -124,9 +125,12 @@ private String getFormId(DelegateTask delegateTask) throws IOException {
                 .collect(Collectors.toList());
 
 
-        if(CollectionUtils.isNotEmpty(userTaskExtensionProperties)) {
+        if (CollectionUtils.isNotEmpty(userTaskExtensionProperties)) {
             String formName = userTaskExtensionProperties.get(0).getCamundaValue();
-            return formSubmissionService.getFormIdByName(StringUtils.substringBefore(getFormUrl(delegateTask),"/form/")+"/"+formName);
+            if (StringUtils.startsWith(formName, "$")) {
```

Comment:
This part of code is already merged. Please update branch with latest base branch